.circleci/config.yml

```diff
@@ -0,0 +1,230 @@
+version: 2.1
```

Comment:
Main changes here. Most of the other changes are in a new Unity project inside `IntegrationTests`